### ISSUE TITLE
Update SR process for Design System Staging Reviews

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#70961]
## User Story
As a Governance team member, I want to hold Staging Review for the Design System team in the same style that Governance holds Staging Reviews for VFS teams so that there's less ambiguity with DST and they're able to easily take action on our findings.

Assignee: @rferguson13 
Peer Reviewer: @allison0034 

## Description
Governance team has held Staging Review with Design System team for 2+ years for changes they are making to Design System components. Instead of aligning our findings to the VA.gov Experience Standards, historically, Design System Staging Reviews have always used the must, should, consider framework. This means that multiple findings may be documented in 1 ticket, and tickets are silo'd across practice area. This makes it more difficult for Design System team to take action, and it also makes it difficult for Governance team to measure and track the findings they have documented with Design System components at Staging Review.

The purpose of this ticket is to pivot our Staging Review process for DST to the process we use with VFS teams. We will document DST findings within the DraftingTemplate, and align them with our revised VA.gov Experience Standards.

The only major difference in a Staging Review for a VFS team vs. a Staging Review for Design System team is that DST Staging Review's will be upheld to a different set of launch-blocking criteria. In fact, the term "launch-blocking" will not be used at all, and instead issues will be classified as high-priority. Since web components are typically live for VFS team use when the DST Staging Review occurs, terms such as "launch-blocking" do not make much sense. Instead, our guidance to DST should notify them which findings should be acted upon immediately (high-priority) and which can be addressed later on (not high-priority). 

## Impacted Artifacts
All documents within the [Design System Staging Reviews](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2572025931/Design+System+Staging+Reviews) folder
DraftingTemplate - create specific version for DST (Shira can handle this)

## Tasks
- [x] Review the 5 documents within the Design System Staging Reviews folder on Confluence. Identify which documents are in need of updating to bring DST up to date with our new process
- [x] Create a Documentation Drafts folder for storing all draft documents for this ticket  
   - Note: since we have not yet gone live with the revised standards, please do not update the current guidance directly. We will create copies to make changes and release later on.
- [x] Review previous DST SR tickets to identify must feedback (equivalent to high-priority findings) and should/consider feedback.
- [x] Generate a list of items that would be considered high-priority vs. low-priority
   - Examples from Matt of high-priority findings:
      - accessibility defects
      - maturity scale being incorrect (has it changed wildly?)
      - does storybook align with design.va.gov?
- [x] Include list of high-priority vs. low-priority in the [How to Document Findings for a Design System Staging Review ](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2572189697/How+to+Document+Findings+for+a+Design+System+Staging+Review)guidance 
- [ ] Anything else?

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] Governance team is prepared to document Staging Review findings for DST in alignment with the new VA.gov Experience Standards
- [ ] Design System team is able to follow the updated guidance for scheduling a SR with Governance

## Team Notification
- [x] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [ ] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by allison0034** [comment]
Since Staging Reviews are the first time seeing DS products, we sometimes have feedback that is not a must. Maybe we just do not label that feedback as "high-priority" and it gets no label and DS team can decide wether or not they want to implement our feedback. Or maybe we add a "consider" label along with "high-priority".

**Comment by shiragoodman** [comment]
@rferguson13 good question. Ideally it will be provided before hand so that we can have the tickets generated going into the meeting. I'm not exactly sure why we don't do that now with DST SRs.

**Comment by rferguson13** [comment]
@allison0034 - Would you mind taking a look at this page: [High priority vs. low priority vs. considerations](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2870149285/70961+-+High+priority+vs.+low+priority+vs.+considerations)? It's not done yet, but I'd like a quick gut check to make sure I'm heading in the right direction.

**Comment by allison0034** [comment]
@rferguson13 I made a couple of small comments. I think this looks really good. I also think as we start using this scale we may tweak this documentation a little.

**Comment by shiragoodman** [comment]
Hey @rferguson13 - I haven't looked at your draft page yet (and I plan to still!) but my only immediate concern right now is the use of "low priority". I'd like if issues were either "high priority", "not high priority" or considerations... similar to what we do with VFS teams (minus considerations)...t's launch-blocking or it's not launch-blocking. I just don't want DST to not end up working on any of our issues bc we flagged them as "low-priority".

Happy to discuss this further if you'd like!

**Comment by rferguson13** [comment]
@shiragoodman - That makes sense to me! I'll change the wording.

**Comment by rferguson13** [comment]
This ticket will continue to the next sprint; I focused more on reviews this week and family visiting. This should be wrapped up quickly in the next sprint.

**Comment by rferguson13** [comment]
Hi @allison0034 I think these are ready for peer review. All the child pages are linked to [this Confluence page](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2870083738/70961+-+Update+SR+process+for+Design+System+Staging+Reviews). 

Waiting on Shira to create the drafting template, so that still needs to be fleshed out. But I figured I'd get you looking at these now. Thanks!

**Comment by allison0034** [comment]
@rferguson13 I made some comments in your drafts. Let me know if we need to chat!

**Comment by shiragoodman** [comment]
@rferguson13 I have made a new [DST DraftingTemplate](https://docs.google.com/spreadsheets/d/1hMcbGxX95gxwEtqxDpJgdDzjeGXSRQxh6iiq9enYp44/edit#gid=727681601) that we can use for DS Staging Reviews. Please take a look and let me know if you have any questions. Thanks!

**Comment by rferguson13** [comment]
Hi @allison0034 - Thank you for your feedback. I've made the updates and added a page. Would you mind taking one last look?

**Comment by rferguson13** [comment]
@shiragoodman - Looks ready to go! Where should I move these drafts to until we're ready for release?

**Comment by shiragoodman** [comment]
@rferguson13 feel free to just leave them where they are in documentation drafts. I plan on talking with Matt tomorrow to see if he wants us to pilot with DST or if we can just go ahead and start this new process. We're both his teams so he gets to call the shots! no other OCTO input needed 😝
